pkgs/development/python-modules/entrypoint2/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildPythonPackage, fetchPypi, EasyProcess, pathpy, pytest }:
+
+buildPythonPackage rec {
+  pname = "entrypoint2";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15dya04884armqjdyqz1jgachkqgh9dc3p25lvyz9afvg73k2qav";
+  };
+
+  propagatedBuildInputs = [ ];
+
+  pythonImportsCheck = [ "entrypoint2" ];
+
+  # argparse is part of the standardlib
+  prePatch = ''
+    substituteInPlace setup.py --replace "argparse" ""
+  '';
+
+  checkInputs = [ EasyProcess pathpy pytest ];
+
+  checkPhase = ''
+    # 0.2.1 has incompatible pycache files included, but this is fixed in master
+    rm -rf tests/__pycache__
```

Comment:
we should make an upstream issue to let them know that their pypi upload wasn't cleaned

Reply:
Done, included in comment.